src/base/net/private/geoipdatabase.cpp

```diff
@@ -221,7 +220,7 @@ bool GeoIPDatabase::parseMetadata(const QVariantHash &metadata, QString &error)
     const QString errMsgNotFound = tr("Metadata error: '%1' entry not found.");
     const QString errMsgInvalid = tr("Metadata error: '%1' entry has invalid type.");
 
-    qDebug() << "Parsing MaxMindDB metadata...";
+    qDebug() << "Parsing  IP DB metadata...";
```

Comment:
Why did you change it? The database is still in MaxMindDB format.

Reply:
@sledgehammer999:  Also, it looks like you accidentally used 2 spaces here, between `Parsing` and `IP`.